src/sentry/incidents/endpoints/validators/numeric_comparison_validator.py

```diff
@@ -8,17 +8,9 @@
 
 
 class NumericComparisonConditionValidator(BaseDataConditionValidator):
-    comparison = serializers.FloatField(
-        required=True,
-        help_text="Comparison value to be compared against value from data.",
-    )
-    condition_result = serializers.ChoiceField(
-        choices=[
-            (DetectorPriorityLevel.HIGH, "High"),
-            (DetectorPriorityLevel.MEDIUM, "Medium"),
-            (DetectorPriorityLevel.LOW, "Low"),
-        ]
-    )
+    type = serializers.ChoiceField(choices=[(t.value, t.value) for t in Condition])
+    comparison = serializers.JSONField(required=True)
+    condition_result = serializers.JSONField(required=True)
```

Comment:
is it possible to have the choices for this be `supported_condition_results`? then we might not need the extra validation function

Reply:
This is for SOLID software design, I actually don't need this line of code at all, it's just explicitly setting the value here the same as the base class to adhere to listkov substitution

Reply:
i see, you are substituting out `validate_condition_result` instead

Reply:
Does not setting it here explciitly violate the principle? One concern I have with including the exact same line here is that if we chance the base class, we need to propagate the same change to a bunch of files. I have the same general concern about all of the duplicated lines.